app/templates/components/follow_button.html

```diff
@@ -0,0 +1,20 @@
+{# Params:
+    - following: Boolean
+    - user: User
+#}
+{% macro follow_button(following, user) -%}
+    {% if following %}
+        {% set css = "btn btn-following" %}
+    {% else %}
+        {% set css = "btn btn-primary" %}
+    {% endif %}
+
+    <button id="follow-button" class="btn {{ css }} shadow-none" onclick="follow(this)"
+            this-user-id="{{ user.user_id }}">
+        {% if following %}
+            Following
+        {% else %}
+            Follow
+        {% endif %}
+    </button>
```

Comment:
Use inline terinaries instead of if statements, simplifies the code a lot

```suggestion
    <button id="follow-button" class="btn {{ 'btn-following' if following else 'btn-primary' }} shadow-none" onclick="follow(this)" this-user-id="{{ user.user_id }}">
        {{ 'Following' if following else 'Follow' }}
    </button>
```